src/main/fc/config.c

```diff
@@ -658,6 +661,16 @@ void saveConfigAndNotify(void)
     beeperConfirmationBeeps(1);
 }
 
+void setConfigDirty(void)
```

Comment:
There are a couple of other places (like ACC calibration) where this needs to be called from -m probably best start with a list of stick commands, and work back from there.

Reply:
The problematic changes are those that are made to the configuration and NOT intended to be saved. Like user makes some adjustments and does not finally decide to save them as a persistent setup. This indication is to allow the stats NOT to save the config with unneeded things (and also the stats from such flight are then not updated). I don't think ACC calibration is problematic here.

Reply:
Doing this half - half and forcing the user to guess if the changes from stick commands will be saved or not is not ideal - we should either auto-save for all, or auto-save for nothing.

Reply:
It's no guess. All stick commands keep working as they did.
The "config dirty" here is for making specific operations (like adjustments, which don't automatically get saved) to work as expected.
Actually ACC calibration from stick commands would also save all configuration, including all unneeded changes from adjustments. The stats code needs special handling because it is executed on every disarm. For ACC it is not that visible because not many would execute ACC calibration after a flight.